mne_bids/tests/test_read.py

```diff
@@ -317,6 +317,32 @@ def test_handle_coords_reading():
     with pytest.raises(RuntimeError, match='Channels do not correspond'):
         raw_test = read_raw_bids(bids_fname, bids_root)
 
+    # make sure montage is set if there are coordinates w/ 'n/a'
+    raw.info['bads'] = []
+    write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
+    electrodes_dict = _from_tsv(electrodes_fname)
+    for axis in ['x', 'y', 'z']:
+        electrodes_dict[axis][0] = 'n/a'
+        electrodes_dict[axis][3] = 'n/a'
+    _to_tsv(electrodes_dict, electrodes_fname)
+
+    # set channel status as 'good' explicitly
+    channels_fname = _find_matching_sidecar(bids_fname, bids_root,
+                                            "channels.tsv",
+                                            allow_fail=True)
+    channels_dict = _from_tsv(channels_fname)
+    channels_dict['status'][0] = 'good'
+    channels_dict['status'][3] = 'good'
+
+    # test if montage is correctly set
+    nan_coord_chs = [channels_dict['name'][i] for i in [0, 3]]
```

Comment:
```suggestion
    nan_chs = [channels_dict['name'][i] for i in [0, 3]]
```
let's be consistent with variable names